packages/toolkit/src/webviews/main.ts

```diff
@@ -216,6 +217,13 @@ export abstract class VueWebview {
                 return this.instance
             }
 
+            public async setup(webview: vscode.Webview) {
```

Comment:
Why call this in advance? Code in this function looks like its handled in the following show() function

Reply:
I cannot call the show() function because that function was specifically design to show a webview in a WebViewPanel as a custom editor on the right hand side. I need to build a function that only takes care of setting up the backend of the vue webview.